executor.sh

```
@@ -6,14 +6,15 @@ echo $DIR
 cd $DIR
 
 mp=m
+package_manager=$(node -p "require('./package.json').packageManager")
 
 sudo ovmlayer create executor
 corepack enable
 
 pnpm install && pnpm build
 tarball=$(cd packages/executor && pnpm pack --pack-destination /tmp/executor)
 tar -zxvf $tarball -C /tmp/executor
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)